kernel.spec

```diff
@@ -483,7 +483,18 @@ if [ "`qubes-prefs -g default-kernel`" == "%version-%plainrel" ]; then
     echo "This kernel version is set as default VM kernel, cannot remove"
     exit 1
 fi
-if qvm-ls --kernel | grep -qw "%version-%plainrel"; then
+
+kernel_used=false
+if qvm-ls --kernel 2>/dev/null >/dev/null; then
+    if qvm-ls --kernel | grep -qw "%version-%plainrel"; then
+        kernel_used=true
+    fi
+else
+    if qvm-ls --raw-data --fields kernel | grep -Fxq "%version-%plainrel"; then
+        kernel_used=true
+    fi
+fi
+if [ "$kernel_used" == true ]; then
```

Comment:
Actually, I was going to backport --kernel option to qvm-ls...

Reply:
Ok, then we can drop this.

Reply:
This has the advantage that it will also cover older (already installed) packages.